### PR TITLE
Backport updates related to https://github.com/devcontainers/devcontainers.github.io/pull/66 to the spec

### PR DESCRIPTION
**Pull request by Chuxel**
Backports https://github.com/devcontainers/devcontainers.github.io/pull/66 to the spec repo.

@bamurtaugh LMK if it would be easier to merge this into https://github.com/devcontainers/spec/pull/128 instead.

### CODE REVIEW

File: docs/specs/devcontainer-features-distribution.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Distribution and Discovery of Dev Container Features
 
+**TL;DR Check out the [quick start repository](https://github.com/devcontainers/feature-template) to get going on distributing your own Dev Container Features.**
```
Comment (bamurtaugh) [inline docs/specs/devcontainer-features-distribution.md:3]: Tiny wording nit :)

```suggestion
**TL;DR Check out the [quick start repository](https://github.com/devcontainers/feature-template) to get started on distributing your own Dev Container Features.**
```

File: proposals/devcontainer-templates-distribution.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Distribution and Discovery of Dev Container Templates
 
+**TL;DR Check out the [quick start repository](https://github.com/devcontainers/template-starter) to get going on distributing your own Dev Container Templates.**
```
Comment (bamurtaugh) [inline proposals/devcontainer-templates-distribution.md:3]:
```suggestion
**TL;DR Check out the [quick start repository](https://github.com/devcontainers/template-starter) to get started on distributing your own Dev Container Templates.**
```

**Review by bamurtaugh — APPROVED** [review]
LGTM! One wording suggestion.

I think go ahead and merge this in - I haven't yet had a chance to resolve the (potentially many) conflicts in the other PR - so might as well get in these changes while there are no conflicts here 😄
